tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3555,3 +3558,55 @@ def combined_non_max_suppression(boxes,
     return gen_image_ops.combined_non_max_suppression(
         boxes, scores, max_output_size_per_class, max_total_size, iou_threshold,
         score_threshold, pad_per_class)
+@tf_export('image.median_filter_2D')
```

Comment:
Our linter will complain about this as we need 2 blank lines before function definitions

Reply:
Solved